### ISSUE TITLE
viva questions

### ISSUE THREAD

**Issue by theorashid** [theorashid/thesis#4]
If the assessors are reading this, please ask the following:

## Chapter 2

> In what applications are nonstationary covariance functions useful?

Lengthscales/variances changes by location.

This failure to adapt to variability, or heterogeneity, in the unknown process is of particular importance in environmental, geophysical, and other spatial datasets, in which domain knowledge suggests that the function may be nonstationary. For example, in mountainous regions, environmental variables are likely to vary much more quickly than in flat regions.

https://onlinelibrary.wiley.com/doi/abs/10.1002/env.785
https://www.stat.berkeley.edu/~paciorek/diss/paciorek-thesis.pdf

> BYM: Are there any problems of assigning priors to the variances of $U$ and $V$?

Yes, the convolution of the random effects components is difficult to fit without strong constraints on one of the two components, as either component can account for most or all of the individual-level variance.

This had let to the BYM2 parametrisation. Rather than a tap for hot and cold, you have a mixer tap with an overall variance and a weighting parameter.

https://mc-stan.org/users/documentation/case-studies/icar_stan.html

> Are spatial models appropriate for the data? They were designed for grids and the spatial units have different sizes

There are concerns that the GMRF representation of space as an adjacency matrix is reductive. Two wildly different geometries can share the same adjacency graph. The ICAR and BYM models were originally proposed for image analysis, where the adjacency matrix for a regular lattice of pixels is more representative of its original form than the graph of a politically- and geographically-influenced boundaries within a country. Further, the ICAR model weights all neighbours equally. Nevertheless, in an epidemiological context, Duncan et al. (2017) compared a BYM model with binary, first-order adjacency matrix to models with a variety of different weighting schemes, including matrix weights based on higher-order degrees of neighbours, distance between neighbours, and distance between covariate values.

Models were consistent when run at district vs aggregate MSOAs to districts – sign that it isn't biased.

https://github.com/athowes/beyond-borders

> What is the Modifiable Areal Unit Problem?

The same basic data yield different results when aggregated in different ways when aggregated in different ways:
- the scale effect, showing major differences depending on the size of units
- the zonation effect, showing major differences depending on how the study area is divided up

Not an issue for me because I did not define MSOAs/LSOAs, and designed to have some homogeneity within them based on household measures. Districts are administrative so local policy happens at that level.

https://athowes.github.io/beyond-borders/resources.html

> What is the difference between indirect and direct standardisation?

Crude mortality rate it $\frac{\text{deaths}}{\text{population}}$. Indirect $\text{SMR} = \frac{\text{actual total number of deaths}}{\text{expected total number of deaths}}$. Direct requires a standard population to compare two sets of age-specific death rates.

https://www.youtube.com/watch?v=zObgZu54mJw

> Why are modern death registration systems, such as that of the UK, _not_ entirely complete and accurate?

Questions on how to count deaths from overseas, deaths of visitors, age at death in older ages, etc. Coroners, slow to add to the dataset for specific causes.

> What was the outcome of the SAHSU studies?

Point source studies:
- Elliot 1992: Clear increase in risk of mesothelioma and asbestosis within 3km of the Plymouth docks.
- Aylin 1999: Small increase in risk but none greater than 10%.
- Hodgson 2004: Significant excess kidney disease near polluting factories in Runcorn.
- Elliot 2001: Landfill sites: small unexplained excess of congenital anomalies, no increase risk of cancers or Down's.

Geographic correlation (physics-informed models to create an exposure surface, and assessed the geographic correlations between this surface and the health outcome):
- Hodgson 2007: plume of mercury pollution
- Elliot 2010: No increased risk for exposure to mobile phone base stations during pregnancy. 
- Hansell 2013: noise from aircraft near Heathrow
- Halonen 2015: Road traffic noise in London
- Parkes 2020: particulate matter from incinerators during pregnancy
> Why did you not cite Bouleri and Blangiardo (2020), Spatio-temporal model to estimate life expectancy and to detect unusual trends at the local authority level in England?

> Do you believe some parts of England are in different stages of the transition to others?

Time trends of different risk factors such as smoking, alcohol use. These risk factors are not to do with infection so probably not still in stage 2.

> What is Malthusian theory?

Population growth is potentially exponential, according to the Malthusian growth model, while the growth of the food supply or other resources is linear, which eventually reduces living standards to the point of triggering a population decline. This event, called a Malthusian check occurs when population growth outpaces agricultural production, causing famine or war, resulting in poverty and depopulation.

> What about smallpox rather than typhoid fever?

Smallpox is also a common example and was actually eliminated through vaccination (last case 1977, WHO 1980), but much later than this reference.

> What does "burden" mean in GBD studies?

Disease burden is the impact of a health problem as measured by financial cost, mortality (YLL), morbidity (YLD), or other indicators.

> What's the difference between an absolute and a compositional model for causes of death?

Absolute models model death rates.

Compositional models relative proportions of each cause of death. Compositional models use an additive logistic normal model (if you apply a multivariate normal model to additive log-ratio it is equivalent to applying a multivariate logistic-normal model to the original compositional dataset). As we are modelling proportions, this does not allow death rates estimation.

> Has anyone jointly modelled the overall death rate and the composition and from those you can estimate the cause-specific death rates which can then be input to the cause-specific likelihoods?

## Chapter 4

> How did you avoid over/underfitting?

The most important thing for us was smoothing. We need death rates between 0 and 1. Followed a Bayesian workflow, starting from a small model and adding effects that we thought were epidemiologically plausible. Effects might not improve model fit and might decrease effective sample size by adding another parameter.

Need sufficient parameters to not have bias at the level of analysis.

Model performance should be evaluated by whether it can predict future data. Use as many methods as you can to justify the model structure - train/test split, argumentation with subject matter experts, WAIC, LOOCV. Given the complexity of the world and the approximate nature of our models, there isn't be “one and preferably only one way” to evaluate how good our model is.

https://discourse.pymc.io/t/do-we-need-a-testing-set/759/5

> What do you classify as a rare event? Do deaths in the oldest age group satisfy this?

“The sample size should be equal to or larger than 20 and the probability of a single success, $p$, should be smaller than or equal to 0.05. If $n$ > 100, the approximation is excellent if $np$ is also < 10.”

Hence why we moved to binomial.

https://www.solon-karapanagiotis.com/post/approx_binomial/approximating-binomial-with-poisson/
https://www.itl.nist.gov/div898/handbook/pmc/section3/pmc331.htm

> Why does INLA scales badly with the number of hyperparameters? What about TMB?

INLA using the Laplace approximation on the latents (hopefully Gaussian) field. Then it performs quadrature (numerical integration) on the hyperparameters, which is slow and hence scales badly.

https://athowes.github.io/thesis/naomi-aghq.html#marginal-la

TMB optimises the hyperparameters so they are fixed values/point estimates. This is Empirical Bayes/type II ML. Kind of like GP where we just optimise for lengthscale/variance, or SSM etc etc. This can underestimate the variances.

> What are the mild conditions for MCMC to converge to the true posterior?

- Irreducibility/ergodicity. It is possible to eventually get from every state to every other state with positive probability
- Aperiodicity. Does _not_ return to the state $i$ after some transition with certainty.
- Stationary/invariant.  If the transition function of that chain leaves that distribution unchanged. Each part of the chain gives the same information: if you split up the chain into parts and rearrange, it should look near-identical.

https://www.youtube.com/watch?v=tByUQbJdt14

> Why did you not use cohort effects like Bennett 2015?

Much longer timescale, forecasting to 2030. 

Time slope for cohort, but not an intercept. Only certain types of age period cohort which are possible due to identifiability.

> How did you choose your priors? What about PC priors?

Started with vague/standard/uninformative priors.

Tested sensitivity to priors (in early models) when first switching from U(0,100) to U(0,2) (vague enough, never near upper limit) to half-normal by comparing scatter plots of death rates with abline.

Now, I would do prior predictive simulation.

**P**enalised **C**omplexity priors (for precisions), which are implemented in INLA but not commonly in most PPLs, penalise departure from the base model such that the probability of the precision above a certain threshold is $\alpha$. So the random effect will not be included unless there is enough information in the data for them to be there. We did not use them because:

- Have to tune the threshold and $\alpha$
- These priors bias the model by reducing variance. That is a modelling choice, and our model has lower bias but more variance.

## Chapter 5

> Figure on probability of dying at different ages. Could there be a survivor effect here? i.e. those in oldest age group have not died young so the SES gradient in the oldest age groups will be weaker than in the younger age groups?

Because they didn't die of this thing earlier, they die of it later so the death rates in older ages might be lower?

> Is there anything similar in European countries? e.g. they have had slowdowns in e0 rises, have the worst off deciles gone into reverse?

Slowdown Leon (2019) paper but not as much as E&W.

"In contrast to the United States, and despite the occurrence of an economic crisis, the health of the lowly educated in Europe has improved in recent years, and health inequalities have sometimes narrowed."
^ Mackenbach 2018. 27 European countries, not UK, trends by education level and by age
https://www.pnas.org/doi/abs/10.1073/pnas.1800028115

## Chapter 6

> What does the beta-binomial likelihood look like? What is $\rho$?

It is a mixture distribution that allows for overdispersion, in the same way the Gamma-Poisson distribution can. The parameter $\rho$ controls the spread of the distribution,

##  Chapter 7

> Why did you group diabetes and nephritis and nephrosis?

Diabetes is the leading cause of kidney disease. Different certifying physicians may assign deaths that have similar natural history to either "diabetes" or "nephritis and nephrosis" depending on their clinical belief on what the underlying cause.

> Why did you use ICAR? Why not use BYM?

We tested a BYM but found it made no difference to fit. ICAR also recommended by Chris Paciorek, statistical help for the project.

Also, the age-space interaction is IID and accounts for a lot of the non-spatial variation, while also allowing for interactions.

> What causes make up the residual groups?

Women: hypertensive heart disease, liver cirrhosis, oesophagus cancer, falls...

Men: pancreatic cancer, bladder cancer, Parkinson's, self-inflicted injuries, stomach cancer, leukaemia...

> Why didn't you use to age 85?

We limited the age range because the probability of death in the absence of competing causes equals 1.0 when the entire life course is considered. We selected 80 years of age as the upper bound because it covers a wide age range but does not include the very oldest ages where multimorbidity makes the assignment of cause of death increasingly difficult.

> Why did you use _unconditional_ probability of dying?

We used unconditional probability of death because removing competing causes of death enhances comparability and equity. For example, two populations with similar exposure to cancer risk factors and similar quality of cancer care, have similar need for effective interventions. If one population has a higher risk of dying from another competing disease, for example HIV or road traffic injuries, the conditional probability would be lower, hence downplaying the need for interventions related to cancers.

> Why didn't you use ASDR?

We used probability of death because it has an intuitive interpretation, and because, unlike age-standardised death rate, it does not depend on the choice of standard population. We also calculated age-standardised death rates. The correlation coefficients between age-standardised death rates and the probability of dying between birth and 80 years of age ranged from 0.93 to >0.99 across sex-year-cancer combinations for the years 2002 and 2019.

> Why are there sex-specific differences in changes for IMPN? Too much smoothing dragging maternal mortality into young ages?

Probably due to a different mix in subcategories. Needs further investigation by separating this category, but difficult to model at a smaller group level. Age at which they die from these causes is young. RW1 is not smooth, similarity.

> Districts with the greatest improvements in life expectancy had large contributions from declines in CVD mortality as well as some of the smallest contributions from increases in mortality from dementias. Was it directional?

> How many moves occur as a short distance?

Van Dijk et al. "the majority of moves occur over short distances: the 2011 Census records that 57.1% of the individuals aged 16 and over that changed address within the preceding 12 months moved within the same local authority district."

However, as discussed in Bennett _et al._ 2023, there might be more between-district migration in London, although this would require further digging into the churn and migration data.

> What does your study add to that already presented in Bennett _et al._ 2018?

- Mapping. Some causes have different spatial patterns, and cannot be attributed only to patterns of deprivation.
- Which causes are driving life expectancy _change_ and their differences between districts?

> Are drug overdoses intentional?

They're listed as intentional on ICD codes.

Although, this is the conservative view that drug use is a personal _choice_. However, there are issues with this related to poverty/low education/poor housing/etc leading to poor mental health/no social support and drug use.

Drug overdose often happens accidentally so would not be "intentional" and same for alcoholic liver disease.

## Chapter 8

> Is there variation between districts of quality of CoD assignment?

- The quality of the cause of death data in the UK has been given the highest rating based on completeness and a low share of deaths assigned to implausible and ill-defined codes. There are national guidelines for death certification, and the Office for National Statistics (ONS) uses standardised coding algorithms and computer-based assignment of the cause of death, which assign the underlying cause of death uniformly so that data are comparable between places and times.
- Validation of cause of death data for prostate cancer in the UK found good sensitivity and specificity.
- Around 30% of deaths go to a coroner. Some evidence of locally issued guidance, but this has been withdrawn since: "Hitherto there have been no such regulations and the circumstances of reporting of deaths by medical practitioners to coroners has varied across coroner areas. As with immediate effect any locally issued guidance or direction should be withdrawn and the principles set out in this document used by all coroners to ensure greater consistency over death reporting."

https://www.ons.gov.uk/peoplepopulationandcommunity/birthsdeathsandmarriages/deaths/methodologies/userguidetomortalitystatisticsjuly2017#cause-of-death-coding

>  Why did you not put CrI for X-fold variation or Spearman correlation?

Presentation. It would result in correlations different to figures which use the median. _But_ we did this for the revisions.

> Colorectal cancer has correlation with poverty of 0.45 for men but only 0.09 for women. Is there an obvious explanation for this?

No obvious explanation (coauthor Amanda Cross). You could imagine that men in general are less likely to seek healthcare for GI symptoms (symptoms associated with later stage disease). This may be exacerbated by SES? Could be related to sex difference in diet.

**Comment by athowes** [comment]
> Why did you use ICAR? Why not use BYM?

Regarding "made no difference" -- is this in outputs? Otherwise, if BYM2, what was the estimated proportion parameter? If BYM, what were the standard deviations for the spatially structured and spatially unstructured parts?

**Comment by theorashid** [comment]
I tried BYM and BYM2 in nimble. I didn't really look at the values of the variances parameters, other than checking they converged. "didn't make a difference" means "didn't improve model fit". Looking at my notes, this is through a mixture of WAIC and plotting the age-specific death rates for different models next to each other.

After that, I stopped using nimble and ported the working model to numpyro
